diagnostic.md

```diff
@@ -27,7 +27,12 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+```$h1-font-size: $base-font-size * 1.3;```
+or
+```h1 {
+      font-size: $base-font-size * 1.3:```
+}
+}
```

Comment:
No, not or. 
```
$h1-font-size: $base-font-size * 1.3
```
defines a new variable called `$h1-font-size`. 
```
h1 {
  font-size: $h1-font-size; // or font-size: $base-font-size * 1.3;
}
```
actually sets the font-size for the h1 elements.